pkg/network/setup/netstat.go

```diff
@@ -32,6 +32,8 @@ import (
 	"kubevirt.io/kubevirt/pkg/virt-launcher/virtwrap/api"
 )
 
+const SRIOVAliasPrefix = "sriov-"
```

Comment:
We cannot reuse it. The soft reason is that we generally should never depend inwards into the kubevirt components from libraries (e.g. depend on virt-launcher code). The hard reasons is that the dependency on virt-launcher hostdevice package brings in dependencies on libvirt dynamic libraries, which do not exist in virt-handler (where this code is used).

I have found no good place to centralize this alias, but if such a place is found, the code can be easily adjusted.

Reply:
We can do the other way around, use SRIOVAliasPrefix from `pkg/network/..` on virt-launcher, WDYT?

Reply:
So your suggestion is to place this constant in the network/setup? It makes no sense to me to have the hostdev/sriov import it from here.
It needs to make sense, and this one does not.

I am also unsure if it fits the network package.

Reply:
Think of this: it makes no sense for hostdev/sriov package to import the whole setup package, when it needs one little constant.

Reply:
I didn't meant that it must be on the setup package, actually I don't mind to which package it will go.
I just prefer to have single place for it like any other constant that is used if few places.
Moreover the `SRIOVAliasPrefix` is our "magic" word that enables finding SR-IOV interfaces on the domain (until we will stop using Hostdev..) thus I think there should be single place for it.
If it means that it should have its own package under pkg/network or somewhere else I think its still better then having it all over.
Having said all that I guess it may bring some difficulties.. then we could do it on a follow up PR.

Reply:
I added a `network/sriov` package.
Done